perun-notification/src/main/java/cz/metacentrum/perun/notif/managers/PerunNotifEmailManagerImpl.java

```diff
@@ -56,272 +39,99 @@ public void init() throws Exception {
 		System.setProperty("client.encoding", "utf-8");
 
 		// Load properties file notif.properties
```

Comment:
`notif.sendMessages` is the last property, which is used from `perun-notif.properties`. I suggest to move it to the `perun.properties`. Then remove all usages of `perun-notif.properties` file. Then we can also remove this comment.

Reply:
We will resolve this in a new pull-request.